src/GraphRenderer.ts

```diff
@@ -1235,8 +1229,10 @@ export class GraphRenderer {
 
         GraphRenderer.dragSelectionHandled(true)
         eagle.isDragging(false);
-        eagle.draggingNode(null)
+        eagle.draggingNode(null);
         
+        //this is a little hack to force affected constructs to update their size on mouse up its neccessary
+        eagle.selectedObjects(eagle.selectedObjects())
```

Comment:
Could we use eagle.selectedObjects.valueHasMutated() here?

Reply:
yep that also works, good call